recipes/sdl/all/CMakeLists.txt

```diff
@@ -2,14 +2,11 @@ cmake_minimum_required(VERSION 3.0)
 project(cmake_wrapper)
 
 include(${CONAN_INSTALL_FOLDER}/conanbuildinfo.cmake)
-conan_basic_setup(KEEP_RPATHS)
+conan_basic_setup(TARGETS)
 
-if(UNIX AND NOT APPLE)
-  if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-    if(CMAKE_CXX_COMPILER_VERSION VERSION_LESS 5.0)
-      add_definitions("-DGBM_BO_USE_CURSOR=2")
-    endif()
-  endif()
+if(APPLE)
+    cmake_minimum_required(VERSION 3.16)
+    enable_language(OBJC)
 endif()
```

Comment:
I don't think this is still required as newer CMake versions apparantly handle Objective C correctly.

Reply:
I revert this type of things to current in CCI